README.md

```diff
@@ -120,6 +120,20 @@ mod ffi {
 }
 ```
 
+## Comparison to other Rust Qt bindings
+
+| project |  integrate into C++ codebase | safe Rust | maintained* | binding mechanism |
+|-------- | ---------------------------- | --------- | ----------- | ----------------- |
+| cxx-qt  |  ✔                           | ✔        |  ✔          | [cxx](https://cxx.rs) plus additional code generation to implement QObject subclasses in Rust and bind them to C++ |
+| [qmetaobject](https://github.com/woboq/qmetaobject-rs/) | ✗ | ✔ | ✔ | [cpp](https://github.com/mystor/rust-cpp) macro to write C++ inline in Rust, plus Rust macros to create QObject subclasses from Rust structs |
+| [Rust Qt binding generator](https://invent.kde.org/sdk/rust-qt-binding-generator) | ✔ | ✔ | ✗ | generates Rust traits and C++ bindings from JSON description of QObject subclass |
+| [rust-qt](https://rust-qt.github.io/) | ✗ | ✗ | ✗ | [ritual](https://rust-qt.github.io/ritual/) to generate unsafe Rust bindings from C++ headers |
+| [qml-rust](https://github.com/White-Oak/qml-rust) | ✗ | ✔ | ✗ | [DOtherSide](https://github.com/filcuc/DOtherSide) C wrapper for QML C++ classes |
+| [qmlrs](https://github.com/flanfly/qmlrs) | ✗ | ✔ | ✗ | own C++ library to bind QQmlApplicationEngine |
+| [qmlrsng](https://github.com/nbigaouette/qmlrsng) | ✗ | ✔ | ✗ | [libqmlbind](https://github.com/seanchas116/libqmlbind) with [bindgen](https://rust-lang.github.io/rust-bindgen/) |
+| [rust-qml](https://github.com/florianjacob/rust-qml) | ✗ | ✔ | ✗ | [libqmlbind](https://github.com/seanchas116/libqmlbind) |
+
+*maintained: repository has had nontrivial commits within last year as of August 2022
```

Comment:
One of those unfortunate things that will make the README look out of date unless we constantly update the date :thinking:

Reply:
yeah... my intention with this is to have some straightforward criterion to classify maintained versus unmaintained

Reply:
Thing is what is "maintained" as you state `rust-qt-binding-generator` as unmaintained, whereas they claim it as "complete". This also creates an arbitrary maintained period for us where CXX-Qt might appear to be unmaintained when it is infact complete.

Reply:
You could say the ritual bindings are "complete" too, but they don't work with Qt6. They don't even build when Qt6 is installed. So I do think "maintained" is a useful column to have in the table.